classes/ActionScheduler_ListTable.php

```diff
@@ -0,0 +1,354 @@
+<?php
+
+/**
+ * Implements the admin view of the actions.
+ */
+class ActionScheduler_ListTable extends PP_List_Table {
+	/**
+	 * The package name. It is used also as the domain for the translations
+	 */
+	protected $package = 'action-scheduler';
+
+	/**
+	 * Columns to show (name => label). The label is automatically
+	 * translated before rendering
+	 */
+	protected $columns = array(
+		'hook'   => 'Hook',
+		'status' => 'Status',
+		'args'   => 'Arguments',
+		'group'  => 'Group',
+		'recurrence' => 'Recurrence',
+		'scheduled'  => 'Scheduled Date',
+		'claim_id'   => 'Claim ID',
+		'comments'   => 'Log',
+	);
+
+	protected $row_actions = array(
+		'hook' => array(
+			'run' => array( 'Run', 'Process the action now as if it were run as part of a queue' ),
+		),
+	);
+
+	/**
+	 *  The active data stores
+	 */
+	protected $store;
+
+	/**
+	 * Bulk actions. The key of the array is the method name of the implementation:
+	 *
+	 *     bulk_<key>(array $ids, string $sql_in).
+	 *
+	 * See the comments in the parent class for further details
+	 */
+	protected $bulk_actions = array(
+		'delete' => 'Delete',
+	);
+
+
+	/**
+	 * If it is true it will load our own javascript library.
+	 *
+	 * Our javascript library will show the logs as a jQuery modal.
+	 */
+	protected static $should_include_js = false;
+
+
+	/**
+	 * Sets the current data store object into `store->action` and initialises the object.
+	 */
+	public function __construct() {
+		self::$should_include_js = true;
+
+		$this->store = (object) array(
```

Comment:
As this property contains two stores, its name should be `$stores` not the singular `$store` to communicate that.

Also, what is the benefit of storing them in a property instead of just calling `ActionScheduler::store()` or  `ActionScheduler::logger()`?

Calling them explicitly makes it clear which store/object is being used when it is used instead of abstracting that information, and it will always be the canonical source of a store, so it's not necessary to future proof it against changes.

Reply:
There is just a minor improvements. It's cheaper to read a property to call a method. But I'll change it.